Program.cs

```diff
@@ -233,6 +233,47 @@ static async Task Main(string[] _)
 
             await ReadyEvent.OnStartup(discord);
             
+            // Migration from joinwatch to user notes
+            var joinWatchedUsersList = await Program.db.ListRangeAsync("joinWatchedUsers");
+            var joinWatchNotesList = await Program.db.HashGetAllAsync("joinWatchedUsersNotes");
```

Comment:
We should check if `joinWatchedUsers` exists first which is a faster op and avoids wasting cycles every startup. If it's empty then you can assume there is no data to migrate and skip fetching the data.